### DISCUSSION TITLE
eslint-config-next not working with eslint v8

### DISCUSSION THREAD

**Discussion by tubbo** [vercel/next.js#34072] [Help] [answered]
When I try upgrading ESLint to v8.8.0, I'm not able to run `eslint .` because of this error coming from `eslint-config-next`:

```
Oops! Something went wrong! :(

ESLint: 8.8.0

Error: Cannot read config file: /[REDACTED]/node_modules/eslint-config-next/index.js
Error: Failed to patch ESLint because the calling module was not recognized.
If you are using a newer ESLint version that may be unsupported, please create a GitHub issue:
https://github.com/microsoft/rushstack/issues
Referenced from: /[REDACTED]/.eslintrc.js
```

We're on Next v11, so we're on `eslint-config-next@11.1.3`. Upgrading to 11.1.4 did not fix the issue.

What's the latest ESLint version that the config supports?

**Comment by dan-made** [comment, marked as answer]
Support for ESLint v8 did not land until [12.0.5](https://github.com/vercel/next.js/releases/tag/v12.0.5), you will need to be on 7.x

**Reply by tubbo** [reply]
ah ok, thanks!
